### PR TITLE
Changed unavailable to in a relation

### PR DESCRIPTION
**Pull request by JeroenED**
I found out that the label which should be used for "in a relation" is not present. However, there are some weird (legacy?) available and unavailable labels.

I think we should remove these in favor of Single and In a relation

Resolves: lots of relationship issues

### CODE REVIEW

**Comment by JeroenED** [comment]
@MrPetovan done!

**Comment by nupplaphil** [comment]
Please have a look at the version counter in `database.sql` too

**Comment by JeroenED** [comment]
@nupplaphil done!

File: update.php [hunk 1]
```diff
@@ -374,3 +374,11 @@ function update_1315()
...
 	DBA::delete('item-delivery-data', ['postopts' => '', 'inform' => '', 'queue_count' => 0, 'queue_done' => 0]);
 }
+
+function update_1318()
+{
+	DBA::update('profile', ['marital' => "In a relation"], ['marital' => "Unavailable"]);
+	DBA::update('profile', ['marital' => "Single"], ['marital' => "Available"]);
+
+	Worker::add(PRIORITY_LOW, 'ProfileUpdate');
+}
```
[the remark this answers is not in the document]
Comment (JeroenED) [reply update.php]: Added to both functions

**Comment by JeroenED** [comment]
the force push is a revert of xgettext. I realized I added strings in as well as #7347 as #7351 which will result in a merge conflict. I'll create an additional pr when both are merged.

File: update.php [hunk 2]
```diff
@@ -373,6 +373,7 @@ function update_1309()
 function update_1315()
 {
 	DBA::delete('item-delivery-data', ['postopts' => '', 'inform' => '', 'queue_count' => 0, 'queue_done' => 0]);
+    return Update::Success;
```
Comment (annando) [inline update.php]: Could you replace the spaces with tabs?

File: update.php [hunk 3]
```diff
@@ -381,4 +382,5 @@ function update_1318()
 	DBA::update('profile', ['marital' => "Single"], ['marital' => "Available"]);
 
 	Worker::add(PRIORITY_LOW, 'ProfileUpdate');
+    return Update::Success;
```
Comment (annando) [inline update.php]: Please replace the spaces here as well.

**Comment by AlfredSK** [comment]
This DB update completely killed my server. I had to revert this PR locally.

**Comment by MrPetovan** [comment]
Sorry about that, any specific error?

**Comment by AlfredSK** [comment]
Hard to say. I had a lot of Apache log errors "database not reachable". When I tried to manually apply the dbstructure update I got the same error: "database not available". I checked the database server. It was running and I could connect to a mysql console but Friendica had opened 150 database connections (my max connections setting). It looked like some loop or something like this.

**Comment by AlfredSK** [comment]
There were two updates to the structure. The one from heluecht (1317) and this one from JeroenED (1318). When I git pulled both updates were attempted to apply. Maybe that was the problem but I don't know for sure.

**Comment by MrPetovan** [comment]
Ok, thank you for the report. There's not much to go on to do an emergency revert, but I'll issue a notice on Friendica Admins.

**Comment by AlfredSK** [comment]
```
PHP Fatal error:  Uncaught Error: Undefined class constant 'Success' in /var/www/html/mordor/update.php:385
Stack trace:
#0 /var/www/html/mordor/src/Core/Update.php(187): update_1318()
#1 /var/www/html/mordor/src/Core/Update.php(139): Friendica\Core\Update::runUpdateFunction(1318, 'update')
#2 /var/www/html/mordor/src/Core/Update.php(52): Friendica\Core\Update::run('/var/www/html/m...')
#3 /var/www/html/mordor/bin/worker.php(36): Friendica\Core\Update::check('/var/www/html/m...', true, Object(Friendica\App\Mode))
#4 {main}
  thrown in /var/www/html/mordor/update.php on line 385
```

**Comment by MrPetovan** [comment]
Thank you very much.

**Comment by AlfredSK** [comment]
I checked that PR out again. One second later the server was dead again.

**Comment by MrPetovan** [comment]
@JeroenED Please check the existence of the class constant and submit the same PR with the necessary changes.

**Comment by MrPetovan** [comment]
> I checked that PR out again. One second later the server was dead again.

Makes sense, the update crashes, so the system starts it again, then it crashes, etc...

**Comment by JeroenED** [comment]
Working on fix. Very easy: Success needs to SUCCESS

Met vriendelijke groet,
----
Jeroen De Meerleer - Heirweg 5G - 9870 Olsene -    https://www.jeroened.be

Mijn excuses, smartphones kunnen mails verzenden

-------- Oorspronkelijk bericht --------
Aan 8 jul. 2019 21:56, Hypolite Petovan schreef:

**Comment by JeroenED** [comment]
Current status of fix: as I know how to resolve the issue, issue needs 5 minutes to solve of which are 4 minutes of opening apps, since time is past sunset, since PR is reverted, I decided to fix tomorrow as this will also decrease amount of possible other fatal blunders.

**Pull request merged**